raws-feed-processor/src/main/config/raws-feed-processor.properties

```diff
@@ -0,0 +1,25 @@
+raws.mesowest.api.key=KzCad2ngouvo0itVwDZDOVajikoA3oaVmzq
```

Comment:
Yes, this needs to change by environment as our requests are tracked by token & restricted if too many requests are made using same token. This is a very good point, i will make sure this is scrubbed.